doc/api/fs.markdown

```diff
@@ -116,6 +116,10 @@ Stop watching for changes on the given `fs.FSWatcher`.
 
 Emitted when the ReadStream's file is opened.
 
+### readStream.path
+
+The path to the file the stream will read from.
```

Comment:
@yorkie I was on the fence about that one, considering the situations where you'd want to check "path", but I'll defer to your opinion since it matches consistency